docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -114,6 +128,13 @@ If your module does not need to target a remote host, you can quickly and easily
 
    $ python library/my_test.py /tmp/args.json
 
+- For the module developed in an existing collection called ``foo.bar``, you may need to do:
```

Comment:
```suggestion
- For a module developed in an existing collection called ``foo.bar``, you may need to do:
```

Reply:
Instead of "foo.bar" everywhere how about changing to something like `my_namespace.my_collection`? It's longer but it's more descriptive.